addons/l10n_hk/data/l10n_hk_chart_data.xml

```diff
@@ -0,0 +1,558 @@
+    </record>
+    <record id="l10n_hk_1130" model="account.account.template">
+        <field name="name">Equipment</field>
+        <field name="code">1130</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1140" model="account.account.template">
+        <field name="name">Decoration</field>
+        <field name="code">1140</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1160" model="account.account.template">
+        <field name="name">Investments</field>
+        <field name="code">1160</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_12" model="account.account.template">
+        <field name="name">Current Assets</field>
+        <field name="code">12</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_current_assets" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1210" model="account.account.template">
+        <field name="name">Petty Cash</field>
+        <field name="code">1210</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_liquidity" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1240" model="account.account.template">
+        <field name="name">Account Receivable</field>
+        <field name="code">1240</field>
+        <field name="reconcile" eval="True" />
+        <field name="user_type_id" ref="account.data_account_type_receivable" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1241" model="account.account.template">
+        <field name="name">Utility &amp; Rental Deposit</field>
+        <field name="code">1241</field>
+        <field name="reconcile" eval="True" />
+        <field name="user_type_id" ref="account.data_account_type_receivable" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_1242" model="account.account.template">
+        <field name="name">Prepayments</field>
+        <field name="code">1242</field>
+        <field name="reconcile" eval="True" />
+        <field name="user_type_id" ref="account.data_account_type_receivable" />
```

Comment:
Why receivable?

Reply:
@rgo-odoo : The type should be current assets

Reply:
Hummm: let's not rush here.
I would say why not receivable.
(I would have it receivable but I am curious what other localization do)

Reply:
@elicoidal I compared with the UK CoA and there are well classified in current assets

Reply:
@lap-odoo @yostashiro I checked the French and Mexican CoA and this type of account is receivable or payable/reconcile=true so IMO deserves a little bit more digging.
A payment received as prepayment or invoice payment for me follows the same logic as for reconciliation and treatment.
Only main difference between Prepayment account and AR account is the position in the BS: a customer prepayment is a liability, not an asset (we received a credit from him without counterpart).

Reply:
@elicoidal We need to know the background of those settings in French and Mexican CoA.  From my experience making these accounts reconcilable could bring extra nuisance with little benefits.  For example:

- You make a HKD500 deposit to Smartone and Smartone deducts HKD88 every months - we may not want to be reconciling these entries.  Just capturing the account balance per partner should suffice.
- You follow Odoo's standard customer deposit flow (i.e. you make an invoice for 50% deposit for a sales order and customer pays, and you make another invoice for the rest once you deliver goods/services), and there is basically no necessity for you to be reconciling the journal items of the deposit account as Odoo automatically creates in the second invoice a counterpart line to the deposit line of the first invoice.

These are just my reasoning and these may be some cases where making these accounts not reconcilable could cause some problems.  Would be nice to get examples of such operation scenarios.

Reply:
@yostashiro 
I provided example for the FR/MX localization for reference and they require context for sure. 
> You make a HKD500 deposit to Smartone and Smartone deducts HKD88 every months - we may not want to be reconciling these entries. Just capturing the account balance per partner should suffice.

This is a specific need that might require to uncheck the box if you do not need reconciliation.

> You follow Odoo's standard customer deposit flow (i.e. you make an invoice for 50% deposit for a sales order and customer pays, and you make another invoice for the rest once you deliver goods/services), and there is basically no necessity for you to be reconciling the journal items of the deposit account as Odoo automatically creates in the second invoice a counterpart line to the deposit line of the first invoice.

Having reconcile =true is the standard behavior in Odoo for this case.

Reply:
@elicoidal 

> Having reconcile =true is the standard behavior in Odoo for this case.

The reason is...?

Reply:
@yostashiro 
This account is a receivable account to manage payments from customers (whether an invoice is already created is irrelevant to the operation).
Usually for this kind of situation we use reconcile = True in Odoo and I agree that there are cases (such as the one that you suggested) that would deserve another set up.

The question is not here: the question is what is the most common case that the localization should cover. Exceptions are handled after the localization is installed via finetuning.

Reply:
@elicoidal I think you took this account as the one that manages customer deposits while me and probably @lap-odoo were talking for prepayments to suppliers (because of the position of this account in CoA).  I think we all agree now that customer deposits should be receivable (and thus reconcile = True).